executor/aggfuncs/func_count_distinct.go

```diff
@@ -493,6 +516,10 @@ func (p *partialResult4ApproxCountDistinct) InsertHash64(x uint64) {
 	p.insertHash(approxCountDistinctHashValue(x))
 }
 
+func (p *partialResult4ApproxCountDistinct) BufLen() int64 {
```

Comment:
I'm not sure whether this is better

BufLen --> BufBytesUsage

Reply:
Maybe we can use `BufUsage`. This is more concise and simple.

Reply:
I modified it to `MemUsage`